bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -200,6 +200,26 @@ def process
                   i18n_strings[BLOCK_CATEGORIES_TYPE][name] = name if name
                 end
 
+                ## Lab2 Function Names.
+                ## Blockly in Lab2 uses JSON rather than XML.
+                if level.uses_lab2?
+                  functions = level.
+                    properties.
+                    dig("level_data", "startSources", "blocks", "blocks")&.
```

Comment:
minor: maybe worth having a helper function on `blockly.rb` like `start_blocks_lab2` or something to share this with other translationc code? but not required.

Reply:
Good call.  Opted to merge but would be a good follow-up.